### ISSUE TITLE
fill-paragraph does not respect comments

### ISSUE THREAD

**Issue by odanoburu** [lijunsong/pollen-mode#22]
if I have a buffer with content like thus

```
◊; lorem
ipsum
```

when I invoke `fill-paragraph` (usually bound to `M-q`), the result is

```
◊; lorem ipsum
```

whereas I would expect the text not to change at all, see the same content in `org` syntax:

```
# lorem
ipsum
```

after pressing `M-q`:

```
# lorem
ipsum
```

**Comment by lijunsong** [comment]
Yes, it seems we need to tune `paragraph-separate` or `paragraph-start` to make it behave correctly. I'll fix it soon. Thanks for reporting.

**Comment by odanoburu** [comment]
I tried this

```
◊; Local Variables:
◊; mode: pollen
◊; paragraph-start: "\\(◊;\\)\\|\\|[ 	]*$"
◊; comment-start: "◊;"
◊; comment-start-skip: "◊;{?[[:space:]]*"
◊; End:
```

and the behaviour improved in some situations, but still doesn't work properly in all of them…
